### ISSUE TITLE
Project Proposal

### ISSUE THREAD

**Issue by noah-springwater** [noah-springwater/Final-Project#1]
## Final Project - Sports Statistics Data Visualization

For my final project I want to create source of data visualization for certain sports statistics. I will explore D3 js as a way of getting outside of my comfort zone to execute the data visualization. As an initial plan, I want the user to be able to select from a series of statistics, which will render a chart or map of the players for a specific position. For example, if I select 'rebounds' for 'point guards' I want the map to render all of the point guards in the league and visually represent their rebounds over the last 5 games.

I want to do a selection of statistics because not only do people enjoy visually represented data, but for people who are interested in fantasy sports, they may want to view how a player is doing compared to other players at a particular position. In attempt to refute psychology's 'hot hand fallacy', in 2013 a NYT article was published (http://well.blogs.nytimes.com/2013/05/08/are-hot-hands-in-sports-for-real/?_r=0), which looked at large data sets of game statistics across various sports. Below is a passage that discusses the data findings: 
```"In these big sets of data, which were far larger than those used in, for instance, the 1985 basketball study, success did slightly increase the chances of subsequent success — though generally over a longer time frame than the next shot. Basketball players experienced statistically significant and recognizable hot periods over an entire game or two, during which they would hit more free throws than random chance would suggest. But they would not necessarily hit one free throw immediately after the last.
Similarly, bowlers who completed a high-scoring game were more likely to roll strikes in the next game. But a strike in one frame of each game was not statistically likely to lead to a strike in the next frame."```

Because of this notion of hot/cold streaks over a longer period time, it will be interesting to visually represent how certain players at certain positions play over a period of time several games.

**Comment by jasonseminara** [comment]
I think this is great! I think you should focus on some specific data points that create a narrative and help tell a story about your hypothesis. Think of this as a thesis, where the data is your research, and the visualizations are your experiments to attempt to prove your thesis. 

TLDR: Narrow down your scope to a select set of data points to complete a fully-functioning website. Build the site in a modular way such that you can come back and add another data set for analysis.

**Comment by jasonseminara** [comment]
Have you thought about narrowing down the scope? Please respond to this before EOD today.
